src/main/java/Duke.java

```diff
@@ -1,10 +1,250 @@
+            markTaskAsDone(index, finishSetup);
+            if (finishSetup) {
+                addInputToFile(input);
+            }
+            break;
+        }
+        case DELETE_COMMAND: {
+            int index = getIndexFromInput(input, command);
+            removeTask(index, finishSetup);
+            if (finishSetup) {
+                addInputToFile(input);
+            }
+            break;
+        }
+        case TODO_COMMAND: {
+            String name = input.substring(TODO_COMMAND.length()).trim();
+            if(name.length() == 0) {
+                throw new EmptyNameException();
+            }
+            Task todo = new Todo(name);
+            addToTaskList(todo, finishSetup);
+            if (finishSetup) {
+                addInputToFile(input);
+            }
+            break;
+        }
+        case DEADLINE_COMMAND: {
+            int byIndex = input.indexOf(BY_COMMAND);
+            if (byIndex == -1) {
+                System.out.println("Please tell me when this is due!");
+                return;
+            }
+            String name = input.substring(DEADLINE_COMMAND.length(), byIndex).trim();
+            String by = input.substring(byIndex + BY_COMMAND.length()).trim();
+            Task deadline = new Deadline(name, by);
+            addToTaskList(deadline, finishSetup);
+            if (finishSetup) {
+                addInputToFile(input);
+            }
+            break;
+        }
+        case EVENT_COMMAND: {
+            int atIndex = input.indexOf(AT_COMMAND);
+            if (atIndex == -1) {
+                System.out.println("Please tell me when is this event!");
+                return;
+            }
+            String name = input.substring(EVENT_COMMAND.length(), atIndex).trim();
+            String at = input.substring(atIndex + AT_COMMAND.length()).trim();
+            Task event = new Event(name, at);
+            addToTaskList(event, finishSetup);
+            if (finishSetup) {
+                addInputToFile(input);
+            }
+            break;
+        }
+        default: {
+            System.out.println("Something went wrong here...");
+            break;
+        }
+        }
+    }
+
+    private static void printList() {
+        System.out.println("\tHere are the tasks in your list:");
+        int numOfTasks = tasks.size();
+        for (int i = 0; i < numOfTasks; i++) {
+            Task task = tasks.get(i);
+            System.out.println("\t" + (i + 1) + ". " + task.toString());
+        }
+    }
+
+    private static int getIndexFromInput(String input, String command)
+            throws NumberFormatException, InvalidIndexException {
+        int index = Integer.parseInt(input.substring(command.length()).trim()) - 1;
+        if (index < 1 || index > tasks.size()) {
+            throw new InvalidIndexException();
+        }
+        return index;
+    }
+
+    private static void markTaskAsDone(int index, boolean print) {
```

Comment:
As with the other booleans with the name `print`, you should consider renaming them to sound more like booleans to follow the coding standard

Reply:
Ok, thank you! :)